src/targetPacks/Directory.Build.props

```diff
@@ -1,6 +1,11 @@
 <Project>
   <Import Project="..\..\Directory.Build.props" />
 
+    <ItemGroup>
```

Comment:
This item group is indented too far.  msbuild artifacts use an indent of 2 spaces instead of 4.

Reply:
Fixed here: https://github.com/dotnet/source-build-reference-packages/pull/794/commits/60266f871372130df00b5cf104002e5951aca8b1#diff-d8109394bde86e6ff38bbfae32c914420f607657372dd94f27448aa406a6e9d8R4